bootstrap/kubeadm/api/v1beta1/kubeadmconfig_types.go

```diff
@@ -75,11 +75,20 @@ type KubeadmConfigSpec struct {
 	// +optional
 	Mounts []MountPoints `json:"mounts,omitempty"`
 
-	// preKubeadmCommands specifies extra commands to run before kubeadm runs
+	// BootCommands specifies extra commands to run very early in the boot process via the cloud-init bootcmd
```

Comment:
FYI during this cycle we started optimizing go doc comments in API packages for usage with kubectl explain, so yaml/json names should be used
```suggestion
	// bootCommands specifies extra commands to run very early in the boot process via the cloud-init bootcmd
```

Same for preKubeadmCommands and postKubeadmCommands
(and same in all the others API packages touched by this PR)